.gitattributes

```diff
@@ -1 +1,5 @@
 * text=auto
+* text eol=crlf
```

Comment:
Why are the text eol=something 2x here? Does it have a technical reason?

Reply:
one is for checkout and one for commit